docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc

```diff
@@ -37,7 +37,7 @@ By default {agent} collects system metrics, such as CPU, memory, network, and fi
       data_stream.dataset: system.filesystem
 -----------------------------------------------------------------------
 
-<1> A generic type describing the data.
+<1> The type of data to input. Refer to <<elastic-agent-input-types>> for the list of what's available.
```

Comment:
It might be just my developer's mind, but it feels a bit odd to me calling this "type of data to input". My understanding is that this is the input name, we use the keyword type but in the end of the day it's the name of the input.

Here it is just slightly different because we also support aliases, which makes thinking about "type of data" even more confusing for me.

Anyway, no need to change anything, I'm just commenting on how I perceive this, my mindset is likely very different than most of our users.

Reply:
That makes sense to me! I've changed the description to `The name of the input.` And after I get the inputs list merged I will add a link to those so people know what values are available.